torch/distributed/__init__.py

```diff
@@ -31,11 +31,6 @@ def init_process_group(backend, init_method='env://', **kwargs):
         rank (int, optional): Rank of the current process.
         group_name (str, optional): Group name. See description of init methods.
     """
-    world_size = kwargs.pop('world_size', -1)
```

Comment:
This part was intentional. We wanted these arguments to be kwarg only, but Py2 doesn't support that by default